lib/resources/r4/appointment_patch.yaml

```diff
@@ -88,3 +88,77 @@ operations:
+    type: string
+    url: https://hl7.org/fhir/R4/datatypes-definitions.html#ContactPoint.value
+    description: The provider video visit link
+    example: |
+      {
+        "op": "replace",
+        "path": "/contained/0/telecom/0/value",
+        "value":  "http://providerlink.vmr.net"
+      }
+    note: >
+      <ul>
+        <li>This patch operation may be used only for video visit appointments</li>
+        <li>This patch operation must be accompanied by all other video visit link patch operations</li>
+        <li>The appointment must already contain VMR data</li>
+      </ul>
+  - name: replace-patient-vmr
+    path: /contained/1/telecom/0/value
+    operation: replace
+    type: string
+    url: https://hl7.org/fhir/R4/datatypes-definitions.html#ContactPoint.value
+    description: The provider video visit link
+    example: |
+      {
+        "op": "replace",
+        "path": "/contained/1/telecom/0/value",
+          "value":  "http://patientlink.vmr.net"
+        }
+    note: >
+       <ul>
+          <li>This patch operation may be used only for video visit appointments</li>
+          <li>This patch operation must be accompanied by all other video visit link patch operations</li>
+          <li>The appointment must already contain VMR data</li>
+        </ul>
+  - name: replace-period-start
+    path: /contained/0/telecom/0/period/start
+    operation: replace
+    type: datetime
+    url: http://hl7.org/fhir/R4/datatypes-definitions.html#Period.start
+    description: The start datetime for the video visit
+    example: |
+      {
+        "op": "replace",
+        "path": "/contained/0/telecom/0/period/start",
+        "value":  "2019-07-13T08:30:00.000Z"
+      }
+    note: >
+      <ul>
+        <li>This patch operation may be used only for video visit appointments</li>
+        <li>This patch operation must be accompanied by all other video visit link patch operations</li>
+        <li>The value for this patch operation will be applied to both the provider video visit link and the patient video visit link</li>
+        <li>The appointment must already contain VMR data</li>
+      </ul>
+  - name: replace-period-end
+    path: /contained/0/telecom/0/period/end
+    operation: replace
+    type: datetime
+    url: http://hl7.org/fhir/R4/datatypes-definitions.html#Period.end
+    description: The end datetime for the video visit
+    example: |
+      {
+        "op": "replace",
+        "path": "/contained/0/telecom/0/period/end",
+        "value":  "2019-07-14T08:30:00.000Z"
+      }
+    note: >
+      <ul>
+        <li>This patch operation may be used only for video visit appointments</li>
+        <li>This patch operation must be accompanied by all other video visit link patch operations</li>
+        <li>The value for this patch operation will be applied to both the provider video visit link and the patient video visit link</li>
+        <li>The appointment must already contain VMR data</li>
+      </ul>
```

Comment:
keep a trailing newline on the file

Reply:
https://github.com/cerner/fhir.cerner.com/pull/385/files#diff-07dbf4d3264a1e7a98f8d854b77f88e9R169